packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -26,26 +26,27 @@ const MenuItems = ({ item, hide_menu_item }) => {
         </BinaryLink>
     );
 };
-
-const MenuLinks = ({ is_logged_in, items, is_pre_appstore }) => (
-    <React.Fragment>
-        {!!items.length && (
-            <div className='header__menu-links'>
-                {items.map(item => {
-                    return (
-                        is_logged_in && (
-                            <MenuItems
-                                key={`${item.icon}${item.id}`}
-                                item={item}
-                                hide_menu_item={item.text() === 'Reports' && is_pre_appstore}
```

Comment:
This is not gonna work for other languages if we hardcode the value of `Reports`, please fix it in your PR.

Reply:
I guess it is handled in this Card - https://redmine.deriv.cloud/issues/80748#Jim/UI_for_Account_setting_is_having_reports_option_for_all_other_languages

Reply:
Not really, it is already being translated in `MenuStore`

Reply:
But with that fix PR, we are checking link_to attribute in  MenuStore. It wouldn’t  be a problem I guess.